scripts/get-packed-versions.sh

```diff
@@ -27,12 +32,24 @@ yarn --silent workspaces info | jq -r '.[].location' | while read -r dir; do
   stem=$(echo "$name" | sed -e 's!^@!!; s!/!-!g;')
   file="$(pwd)/${stem}-v${version}.tgz"
 
+  # Clean up.
+  rm -f "${stem}"-v*.tgz
+
   # Create the tarball.
   yarn pack 1>&2
 
+  if $cache_bust; then
+    # Bust the cache!
```

Comment:
Why not do this unconditionally?

Reply:
I'd be happy to make that change.  Will do before landing.